src/main/python/org/broadinstitute/hellbender/gcnvkernel/structs/metadata.py

```diff
@@ -15,7 +15,7 @@ class IntervalListMetadata:
         interval_list: the interval list itself
         num_intervals: number of intervals in the list
         contig_set: set of contigs spanned by the intervals
-        contig_list: sorted list of contigs
+        ordered_contig_list: list of contigs in the order that appears in the interval list
```

Comment:
It is actually the interval list.`gcnvkernel` does not parse the seq dictionary. The orders must match anyway since GATK validates the order of the interval list w/ its SAM seq dict.